plugins/CoreHome/angularjs/reporting-page/reportingpage.controller.js

```diff
@@ -35,6 +41,73 @@
             $scope.renderPage($search.category, $search.subcategory);
         }
 
+        function showOnlyRawDataMessageIfRequired() {
+            if (hasRawData && hasNoVisits) {
+                $scope.showOnlyRawDataAlert = true;
+            }
+
+            var $search = $location.search();
+
+            if ($search.segment !== '') {
+                $scope.showOnlyRawDataAlert = false;
+                return;
+            }
+
```

Comment:
The original issue says:

> Maybe we would only do this extra API call if the date includes "today" as in other cases it should not be needed.

So guess another check for this is missing here. Or is it supposed to be displayed for all dates/periods?

Reply:
yes, that's still missing, I will add.